bullet_train/app/helpers/account/dates_helper.rb

```diff
@@ -1,40 +1,24 @@
 module Account::DatesHelper
-  # e.g. October 11, 2018
-  def display_date(timestamp, custom_date_format = nil)
+  def display_date(timestamp, format: nil)
     return nil unless timestamp
-    if custom_date_format
-      local_time(timestamp).strftime(custom_date_format)
-    elsif local_time(timestamp).year == local_time(Time.now).year
-      local_time(timestamp).strftime("%B %-d")
-    else
-      local_time(timestamp).strftime("%B %-d, %Y")
-    end
+    timestamp = set_to_user_time_zone(timestamp)
+    formatted_timestamp(timestamp, format: format)
   end
+  alias_method :display_date_and_time, :display_date
 
-  # e.g. October 11, 2018 at 4:22 PM
-  # e.g. Yesterday at 2:12 PM
-  # e.g. April 24 at 7:39 AM
-  def display_date_and_time(timestamp, custom_date_format = nil, custom_time_format = nil)
-    return nil unless timestamp
-
-    # today?
-    if local_time(timestamp).to_date == local_time(Time.now).to_date
-      "Today at #{display_time(timestamp, custom_time_format)}"
-    # yesterday?
-    elsif (local_time(timestamp).to_date) == (local_time(Time.now).to_date - 1.day)
-      "Yesterday at #{display_time(timestamp, custom_time_format)}"
-    else
-      "#{display_date(timestamp, custom_date_format)} at #{display_time(timestamp, custom_time_format)}"
-    end
-  end
+  private
 
-  # e.g. 4:22 PM
-  def display_time(timestamp, custom_time_format = nil)
-    local_time(timestamp).strftime(custom_time_format || "%l:%M %p")
+  # If the format is a Symbol, we use the standard Rails formatting.
+  # https://guides.rubyonrails.org/i18n.html#adding-date-time-formats
+  def formatted_timestamp(timestamp, format: nil)
+    format ||= :default
+    format.is_a?(Symbol) ? l(timestamp, format: format) : timestamp.strftime(format)
   end
 
-  def local_time(time)
```

Comment:
Since this is a public method is there any chance that people are relying on the old method name? I.e. can we just yank it?

Reply:
Yeah you're 100% right, I added the method back in case other developers are using it.